models/Agriculture_price_prediction/app.py

```diff
@@ -0,0 +1,259 @@
+
+        .main {
+            background-image: url("https://wallpapercave.com/wp/wp9212753.jpg");
+            background-size: cover;
+            background-attachment: fixed;
+            background-repeat: no-repeat;
+            background-position: center;
+            position: relative;
+            font-family: 'Roboto', sans-serif;
+        }
+        .overlay {
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            background: rgba(255, 255, 255, 0.7); /* Slightly more transparent overlay */
+            z-index: 1;
+        }
+        .sidebar .sidebar-content {
+            background-color: rgba(255, 255, 255, 0.9); /* Solid background for sidebar */
+            padding: 15px;
+            border-radius: 12px;
+            border: 1px solid #ddd;
+            box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+        }
+        .stButton button {
+            background-color: #007BFF;
+            color: white;
+            border-radius: 8px;
+            padding: 12px;
+            font-weight: 500;
+            border: none;
+        }
+        .stButton button:hover {
+            background-color: #0056b3;
+        }
+        h1 {
+            color: #003366;
+            text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.3);
+            position: relative;
+            z-index: 2; /* Ensure text is above overlay */
+        }
+        h3 {
+            color: #333333;
+            text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
+            position: relative;
+            z-index: 2; /* Ensure text is above overlay */
+        }
+        footer {
+            color: #555555;
+            text-align: center;
+            padding: 15px 0;
+            font-size: 14px;
+            border-top: 1px solid #ddd;
+            position: relative;
+            z-index: 2; /* Ensure text is above overlay */
+        }
+        .stats-box {
+            background-color: rgba(255, 255, 255, 0.9); /* Solid background for statistics */
+            padding: 15px;
+            border-radius: 12px;
+            border: 1px solid #ddd;
+            box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+            margin: 20px 0;
+        }
+    </style>
+    """,
+    unsafe_allow_html=True
+)
+
+# Add an overlay div
+st.markdown('<div class="overlay"></div>', unsafe_allow_html=True)
+
+# Title with gradient and shadow
+st.markdown("<h1 style='text-align: center;'>Agricultural Price Forecasting</h1>", unsafe_allow_html=True)
+st.markdown(
+    "<h3 style='text-align: center;'>Predict and Visualize Commodity Prices Based on Rainfall and Temperature</h3>",
+    unsafe_allow_html=True)
+
+# Sidebar for user inputs with better organization
+st.sidebar.markdown("<h2 style='color: #0056b3;'>User Inputs</h2>", unsafe_allow_html=True)
+
+with st.sidebar.expander("Select Parameters", expanded=True):
+    state = st.selectbox("Select State", ["Mumbai"],
```

Comment:
### 🔴 Step 6:
Only a single state is available, see If you have any unused files, or options. Or maybe you plan on working on them later on.

Make the changes accordingly.